centreon/www/class/centreonGraph.class.php

```diff
@@ -389,7 +398,7 @@ public function init()
      */
     private static function quote($elem)
     {
-        return "'" . $elem . "'";
+        return $elem ;
```

Comment:
Changing this could cause breaking change

```suggestion
        return "'" . $elem . "'";
```

Reply:
i test it no brecking change 
i did that to update a SQL query because it Doesn't work because the 'bindValue' function will enclose the value in single quotes.

Reply:
i test it no breaking change
i did it because the 'bindValue' function will enclose the value in single quotes.